static/login.html

```diff
@@ -14,40 +11,32 @@
       .button {
         border-radius: 2px;
       }
+
+      .note {
+        font-size: 0.7rem;
+        text-decoration: italic;
+        padding-top: 1rem;
+      }
     </style>
   </head>
   <body>
-    <div class="main-body">
-      <main>
-      <h1>
-        Login
-      </h1>
-      <h5>
-        Please use charlie@gmail.com to test a successful login.
-        <h5>
+    <main class="main-body">
+      <h1>Login</h1>
+      <p>
+        Use mark@gmail.com or jane@gmail.com (power user) with any password
+        to perform a successful login.
+      </p>
       <form method="post" action="submit" id="login-form">
-          <div class="label">Email:</div>
-          <input type="email" name="email" value="charlie@gmail.com">
+        <div class="label">Email:</div>
+          <input type="email" name="email" value="mark@gmail.com">
         <div class="form-line">
           <div class="label">Password:</div>
-          <input type="password" name="password" value="123456">
+          <input type="password" name="password" value="">
         </div>
-          <input name="returnurl" type="hidden" value="{{.ReturnURL}}">
-          <input name="token" id="token" type="hidden" value="">
-          <input type="submit" value="Login" class="button">
+        <input name="returnurl" type="hidden" value="{{.ReturnURL}}">
+        <input type="submit" value="Login" class="button">
       </form>
-      <p> Or you can use Google Sign in </p>
```

Comment:
The login page is also used here: 
https://ampbyexample.com/samples_templates/comment_section/, so you'd need to change that page as well. I'd prefer to keep it though until we have a dedicated sample.

Reply:
I'm aware of it, that sample remains fully functional with this PR.

Reply:
Doesn't it use the same login page?

Reply:
It does, but this PR doesn't break it, i.e. you can still use the same login page and the login will still work on the comment section sample.

Reply:
yes, but the sample doc mentions google signin

Reply:
Ouch, missed that, good point.